Makefile

```diff
@@ -135,13 +135,16 @@ deploy-with-olm: kustomize ## Deploy controller to the K8s cluster via OLM
 undeploy-with-olm: ## Undeploy controller from the K8s cluster
 	$(KUSTOMIZE) build config/install | kubectl delete -f -
 
+OPERATOR_TYPE ?= standalone
```

Comment:
As we know this is a standalone operator only why can not we directly add the annotation in the CSV instead of building automation around it?

If we really need this, Lets add this variable in the `hack/make-bundle-vars.mk`

Reply:
I've been switching it to `non-standalone` for a few tests and experiments. Though I agree, this would be better in the `.mk` files.